features/step_definitions/cars_garage_steps.rb

```diff
@@ -27,6 +27,7 @@
 
 When("I submit filled in new car form") do
   fill_in 'car_custom_name', with: "Test car custom name"
+  sleep 2
```

Comment:
Mind ⚠️

Reply:
"How to speed up your app? Just remove `sleep` you've put in code before" :)